doc/VtExplain.md

```diff
@@ -0,0 +1,143 @@
+# VTExplain Tool
+
+The vtexplain tool provides information about how Vitess will execute a statement (the Vitess version of MySQL "EXPLAIN").
+
+## Prerequisites
+
+You'll need to build the `vtexplain` binary in your environment.
+To find instructions on how to build this binary please refer to this [guide](http://vitess.io/getting-started/local-instance.html#manual-build). 
```

Comment:
We are currently in a state where most doc/*.md files are included on www.vitess.io, but not all of them. This file is an example which is not included on www.vitess.io.

IMO we should change this and remove the possibility that people accidentally read the documentation on GitHub in the "doc" folder instead of on www.vitess.io.

A consequence of that change would be that you shouldn't use Markdown to link other internal pages. Instead, the function from the used HTML generator Jekyll should be used: http://vitess.io/internal/publish-website/#linking-pages

Also note that including pages on www.vitess.io and building it will become much simpler very soon with the integration of Netlify.com. For example, you can see a preview of your changes when you create the pull request.

Tagging @sougou and @alainjobart here.

Should this page go into the "User Guide" section or should we start to create more high-level sections to not overcrowd "User Guide"?

Reply:
I've struggled with this issue a bit also. Ideally, links should work whether you're reading md files or vitess.io. Is there a way to auto-translate?

I think it's kind of good that md files are here. There may be people who are landing directly here. But I could be convinced to remove the md files if there's no way to make them work for both situations.

Reply:
Unfortunately, there's no way to auto translate between these two formats.

Therefore, I think we should purge the "doc/" directory and only leave a README.md which points to www.vitess.io. This will avoid the confusion that you can read these pages on GitHub or on www.vitess.io.

Reply:
I agree with @michael-berlin, vitess.io should be the main place to read the docs. Once we finish the Netlify migration, my vote is to get rid of `/doc` and `/docs` from the repo.

Reply:
Sounds good from my perspective. Let me know when is the right time and I can move it. Also if you have instructions on how to add docs to vitess.io, that would be great!